src/components/SpectralProfiler/SpectralProfilerSettingsPanelComponent/SpectralProfilerSettingsPanelComponent.tsx

```diff
@@ -8,14 +8,16 @@ import {SpectralProfileWidgetStore} from "stores/widgets";
 import {WidgetProps, DefaultWidgetConfig, HelpType, AppStore, WidgetsStore} from "stores";
 import {parseNumber} from "utilities";
 import "./SpectralProfilerSettingsPanelComponent.scss";
+import { ProfileFittingComponent } from "../ProfileFittingComponent/ProfileFittingComponent";
```

Comment:
inconsistent whitesapce. Also please make sure the `.scss` import is last

Reply:
modified. thanks for the remind of the `.scss` import